### ISSUE TITLE
loss累积写法 可能会导致显存溢出

### ISSUE THREAD

**Issue by CarryHJR** [bubbliiiing/yolo3-pytorch#3]
train.py中 total_loss += loss 和val_loss += loss，这种写法是pytorch 0.4之前的写法，0.4之后要改为
+= loss.item()，可参考https://zhuanlan.zhihu.com/p/59271905， 如果batchsize设的小的话可能遇不到这个错误
